src/dmd/frontend.h

```diff
@@ -171,9 +171,8 @@ enum class LINK : uint8_t
     c = 2u,
     cpp = 3u,
     windows = 4u,
-    pascal = 5u,
-    objc = 6u,
-    system = 7u,
+    objc = 5u,
```

Comment:
Oops, just realised that you broke downstream gdc and ldc.

Reply:
Isn't it expected tho ?

Reply:
> Isn't it expected tho ?

You didn't update global.h

Reply:
Ergh, an oversight, will fix.